docs/reference/setup/install/docker.asciidoc

```diff
@@ -27,40 +40,133 @@ endif::[]
 ifeval::["{release-state}"!="unreleased"]
 
 [source,sh,subs="attributes"]
---------------------------------------------
+----
 docker pull {docker-repo}:{version}
---------------------------------------------
+----
 
 endif::[]
 
+Now that you have the {es} Docker image, you can start a 
+<<docker-cli-run-dev-mode,single-node>> or <<docker-compose-file,multi-node>>
+cluster.
+
 [[docker-cli-run-dev-mode]]
-==== Starting a single node cluster with Docker
+==== Start a single-node cluster with Docker
 
 ifeval::["{release-state}"=="unreleased"]
 
 WARNING: Version {version} of the {es} Docker image has not yet been released.
 
 endif::[]
 
-ifeval::["{release-state}"!="unreleased"]
+If you're starting a single-node {es} cluster in a Docker container, security
+will be automatically enabled and configured for you. When you start {es} for
+the first time, the following security configuration occurs automatically:
+
+* Certificates and keys are generated for the transport and HTTP layers.
+* The Transport Layer Security (TLS) configuration settings are written to
+`elasticsearch.yml`.
+* A password is generated for the `elastic` user.
+* An enrollment token is generated for {kib}.
+
+The following certificates and keys are generated in the
+`/usr/share/elasticsearch/config/tls_auto_config_initial_node_<timestamp>`
+directory, and allow you to connect a {kib} instance to your secured {es}
```

Comment:
"The following certificates and keys are generated in the
`/usr/share/elasticsearch/config/tls_auto_config_initial_node_<timestamp>`
directory in the docker container..."